### PR TITLE
chore: [TECH-1317] migrate tooltip to dhis2/ui

### PR DESCRIPTION
**Pull request by jasminenguyennn**

### CODE REVIEW

File: src/core_modules/capture-core/components/ListView/ColumnSelector/ColumnSelector.component.js [hunk 1]
```diff
@@ -45,10 +45,8 @@ export class ColumnSelector extends React.PureComponent<Props, State> {
         return (
             <React.Fragment>
                 <Tooltip
-                    disableFocusListener
-                    disableTouchListener
-                    enterDelay={500}
-                    title={i18n.t('Select columns')}
+                    openDelay={500}
+                    content={i18n.t('Select columns')}
```
Comment (jasminenguyennn) [inline src/core_modules/capture-core/components/ListView/ColumnSelector/ColumnSelector.component.js:49]: Screenshot 2022-08-19 at 13 30 08

File: src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentQuickActions/QuickActionButton/QuickActionButton.js [hunk 2]
```diff
@@ -14,20 +14,30 @@ const styles = {
 };
 
 const QuickActionButtonPlain = ({ icon, label, onClickAction, dataTest, disable, classes }: QuickActionButtonTypes) => (
-    <Button
-        onClick={onClickAction}
-        dataTest={dataTest}
-        disabled={disable}
+    <Tooltip
+        content={i18n.t('No available program stages')}
```
Comment (jasminenguyennn) [inline src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentQuickActions/QuickActionButton/QuickActionButton.js:18]: Screenshot 2022-08-19 at 13 52 18

File: src/core_modules/capture-core/components/Pages/ViewEvent/EventDetailsSection/EventDetailsSection.component.js [hunk 3]
```diff
@@ -94,20 +94,29 @@ const EventDetailsSectionPlain = (props: Props) => {
...
+                <div
+                    className={classes.editButtonContainer}
+                >
+                    <Button
+                        className={classes.button}
+                        variant="raised"
+                        onClick={() => onOpenEditEvent(orgUnit)}
+                        disabled={!canEdit}
+                    >
+                        <Tooltip content={i18n.t('You don\'t have access to edit this event')}>
```
Comment (jasminenguyennn) [inline src/core_modules/capture-core/components/Pages/ViewEvent/EventDetailsSection/EventDetailsSection.component.js:106]: Screenshot 2022-08-19 at 13 51 42

File: src/core_modules/capture-core/components/Relationships/Relationships.component.js [hunk 4]
```diff
@@ -133,15 +133,24 @@ class RelationshipsPlain extends React.Component<Props> {
                     </div>
                 </div>
                 <div className={classes.relationshipActions}>
-                    <Tooltip title={canDelete ? '' : i18n.t('You dont have access to delete this relationship')}>
-                        <div>
-                            <IconButton
-                                onClick={() => { onRemoveRelationship(relationship.clientId); }}
-                                disabled={!canDelete}
+                    <Tooltip content={i18n.t('You don\'t have access to delete this relationship')}>
```
Comment (jasminenguyennn) [inline src/core_modules/capture-core/components/Relationships/Relationships.component.js:136]: Screenshot 2022-08-19 at 14 01 12

File: src/core_modules/capture-core/components/WidgetEnrollment/Actions/AddNew/AddNew.component.js [hunk 5]
```diff
@@ -4,26 +4,32 @@ import React from 'react';
...
-    {...passOnProps}
-/>);
-
 export const AddNew = ({ canAddNew, onlyEnrollOnce, tetName, onAddNew }: Props) => {
     if (!canAddNew) {
         return null;
     }
 
-    return (onlyEnrollOnce ? <Tooltip
+    return (<Tooltip
```
Comment (jasminenguyennn) [inline src/core_modules/capture-core/components/WidgetEnrollment/Actions/AddNew/AddNew.component.js:12]: Screenshot 2022-08-19 at 13 29 50

File: src/core_modules/capture-core/components/DataTable/SortLabelWrapper.component.js [hunk 6]
```diff
@@ -74,9 +73,9 @@ class SortLabelWrapperPlain extends React.Component<Props> {
 
         return (
             <Tooltip
-                title={i18n.t('Sort')}
+                content={i18n.t('Sort')}
```
Comment (jasminenguyennn) [inline src/core_modules/capture-core/components/DataTable/SortLabelWrapper.component.js:76]: Screenshot 2022-08-19 at 14 08 35

**Review by simonadomnisoru — APPROVED** [review]
LGTM! The print screens were very helpful.

**Review by geethaalwan — APPROVED** [review]
Tested successfully on 2.39 version

**Pull request merged**